test/docker-compose.yml

```diff
@@ -27,3 +27,9 @@ services:
     extends:
       service: base_image
     command: "bash -c 'cd /go/src/github.com/cilium/cilium/; make precheck || exit 1'"
+  publish_unit_test_cover:
+    extends:
+      service: base_image
+    environment:
+      - "COVERALLS_TOKEN"
+    command: "bash -c 'cd /go/src/github.com/cilium/cilium/; go get golang.org/x/tools/cmd/cover && go get github.com/mattn/goveralls && goveralls -coverprofile=coverage-all.out -service=travis-ci -repotoken=${COVERALLS_TOKEN}'"
```

Comment:
TBH, the idea of using Docker-compose is to have the use of a docker-image and do not install in each build that tools.

Reply:
this is still WIP, I'll create an image once I know this is working